shared/src/business/entities/cases/PublicCase.isPrivateDocument.test.ts

```diff
@@ -1,155 +1,343 @@
-      {
-        docketEntryId: '1451c228-49d6-44a9-ac02-d797be308661',
-        documentType: 'Amended',
-        eventCode: 'AMAT',
-        filingDate: '2000-12-02T05:00:00.000Z',
-        isOnDocketRecord: true,
-        isStricken: false,
-        previousDocument: {
-          docketEntryId: '2a389787-91d2-41ba-9c6e-2a13440a928b',
-          documentType: 'Seriatim Answering Brief',
+              expect(isPrivate).toEqual(false);
+            },
+          );
+
+          it.each(otherRoles)(
+            'returns true if the filing party is not a practitioner and it is electronically filed',
+            filedByRole => {
+              const isPrivate = PublicCase.isPrivateDocument(
+                {
+                  ...baseDocketEntry,
+                  filedByRole,
+                },
+                visibilityChangeDate,
+              );
+
+              expect(isPrivate).toEqual(true);
+            },
+          );
         },
-      },
-      visibilityChangeDate,
-    );
+      );
 
-    expect(isPrivate).toEqual(true);
-  });
+      describe.each(amendmentEventCodes)(
+        'Amendments to Practitioner-Filed Briefs',
+        eventCode => {
+          // must be electronic
+          // must be filed by Practitioner
+          const baseDocketEntry = {
+            docketEntryId: '123',
+            eventCode,
+            filedByRole: 'privatePractitioner',
+            filingDate,
+            isOnDocketRecord: true,
+            isPaper: false,
+            previousDocument: {
+              docketEntryId: '2a389787-91d2-41ba-9c6e-2a13440a928b',
+              documentType: 'Seriatim Answering Brief',
+            },
+          };
+
+          it('returns true if it is paper', () => {
+            const isPrivate = PublicCase.isPrivateDocument(
+              {
+                ...baseDocketEntry,
+                isPaper: true,
+              },
+              visibilityChangeDate,
+            );
+
+            expect(isPrivate).toEqual(true);
+          });
+
+          it.each(practitionerRoles)(
+            'returns false if the filing party is a practitioner and it is electronically filed',
+            filedByRole => {
+              const isPrivate = PublicCase.isPrivateDocument(
+                {
+                  docketEntryId: '123',
+                  eventCode,
+                  filedByRole,
+                  filingDate,
+                  isOnDocketRecord: true,
+                  isPaper: false,
+                  previousDocument: {
+                    docketEntryId: '2a389787-91d2-41ba-9c6e-2a13440a928b',
+                    documentType: 'Seriatim Answering Brief',
+                    // filedByRole: 'privatePractitioner'
+                  },
+                },
+                visibilityChangeDate,
+              );
+
+              expect(isPrivate).toEqual(false);
+            },
+          );
+
+          // there's currently no good way to determine this information
```

Comment:
❓ : Are these test scenarios going to be addressed or can they be deleted?

Reply:
they are going to have to be addressed, but not in the scope of this pr